Makefile

```diff
@@ -13,37 +12,37 @@ all: lint install
 
 ldflags = -X github.com/icon-project/centralized-relay/cmd.Version=$(VERSION) \
 					-X github.com/icon-project/centralized-relay.Commit=$(COMMIT) \
-					-X github.com/icon-project/centralized-relay.Dirty=$(DIRTY)
+					-X github.com/icon-project/centralized-relay.Dirty=$(DIRTY) \
+					-s -w
 
 ldflags += $(LDFLAGS)
 ldflags := $(strip $(ldflags))
 
-BUILD_FLAGS := -ldflags '$(ldflags)'
+BUILD_FLAGS := -mod=readonly -trimpath -ldflags '$(ldflags)'
 
 build: go.sum
-ifeq ($(OS),Windows_NT)
-	@echo "building centralized-relay binary..."
-	@go build -mod=readonly $(BUILD_FLAGS) -o build/centralized-relay main.go
-else
 	@echo "building centralized-relay binary..."
-	@go build  $(BUILD_FLAGS) -o build/centralized-relay main.go
-endif
+	ifeq ($(OS),Windows_NT)
+		@go build $(BUILD_FLAGS) -o build/centralized-relay main.go
+	else
+		@go build $(BUILD_FLAGS) -o build/centralized-relay main.go
+	endif
 
 build-docker:
 	@echo "building centralized docker image..."
 	docker build . -t centralized-relay
 
 install: go.sum
-	@echo "installing centralized-relay binary..."
-	@go build -mod=readonly $(BUILD_FLAGS) -o $(GOBIN)/centralized-relay main.go
+	@echo "--> Installing centralized-relay binary to $(GOBIN)"
+	@go build $(BUILD_FLAGS) -o $(GOBIN)/centralized-relay main.go
 
 
 e2e-test:
 	@go test -v ./test/e2e -testify.m TestE2E_all
 
 PACKAGE_NAME          := github.com/icon-project/centralized-relay
-GOLANG_CROSS_VERSION  ?= v1.22.3
-LIBWASM_VERSION 		 ?= v2.0.0
+GOLANG_CROSS_VERSION  ?= v1.22.4
+LIBWASM_VERSION 		 ?= v2.0.1
```

Comment:
Can we create separate PR for different changes instead of packaging every changes in a single PR ?